plugin/NotificationServiceExtension/AirshipNotificationService.swift

```diff
@@ -0,0 +1,8 @@
+//  NotificationService.swift
+
+
+import AirshipServiceExtension
+
+class AirshipNotificationService: UANotificationServiceExtension {
```

Comment:
Could you make it possible for them to give us a file path to use instead of ours?

Reply:
Yeah I can do that